specs/CookieManagement.md

```diff
@@ -461,7 +465,7 @@ namespace Microsoft.Web.WebView2.Core
 
         /// The expiration date and time for the cookie as the number of seconds since the UNIX epoch.
         /// The default is -1.0, which means cookies are session cookies by default.
-        Double Expires { get; set; };
+        Windows.Foundation.DateTime Expires { get; set; };

```

Comment:
Does this work in WPF/Winforms? I believe `Windows.Foundation.DateTime` is UWP only, isn't it? In case it is, then we should revert back to `Double`

Reply:
We use `Windows.Foundation.DateTime` for WinRT API only, and we use Double for .NET API.

Reply:
There's a datetime type in .NET also right? Can't we use the appropriate datetime type for .NET?